DCOS-46094 COPS-3907

TattdCodeMonkey opened the pull request:
- See #3454 

## Testing
- See [COPS-3907](https://jira.mesosphere.com/browse/COPS-3907)

nLight commented:
Please update i18n

GeorgiSTodorov commented:
Steps to update translations: https://wiki.mesosphere.com/display/ENGINEERING/Extract+and+update+translation+messages .

TattdCodeMonkey commented:
@nLight I rebased and updated. I had to ensure I was on updated `master` for my plugins directory. But *then* when I ran lingui extract script there were no changes. Which is what I was originally expecting with this change. The only code file changes were additions, 1) to an existing line and 2) to the end of the file. So that shouldn't cause any updates to `messages.json`
